gnovm/stdlibs/time/timezoneinfo.gno

```diff
@@ -4,11 +4,7 @@
 
 package time
 
-import (
-	"errors"
-	// "sync" XXX
-	// "syscall" XXX
```

Comment:
We want to keep these, to indicate what changed with upstream go.

Reply:
[2f26e54](https://github.com/gnolang/gno/pull/3193/commits/2f26e5418f7c7da6701ec2b787523b3140055f2f)